tests/integration/test_commands.py

```diff
@@ -1,20 +1,1289 @@
+                sex_validation = dv
+                continue
+            elif f"{columns['Diagnosis']}2" in dv.sqref:
+                diagnosis_validation = dv
+                continue
+            elif f"{columns['Cancer Type']}2" in dv.sqref:
+                cancer_type_validation = dv
+                continue
+            # AND there are no other data validations
+            assert False, f"Unexpected data validation found: {dv}"
+
+        assert sex_validation is not None
+        assert sex_validation.type == "list"
+        assert sex_validation.formula1 == "Sheet2!$B$2:$B$4"
+
+        assert diagnosis_validation is not None
+        assert diagnosis_validation.type == "list"
+        assert diagnosis_validation.formula1 == "Sheet2!$D$2:$D$3"
+
+        assert cancer_type_validation is not None
+        assert cancer_type_validation.type == "list"
+        assert cancer_type_validation.formula1 == "Sheet2!$F$2:$F$6"
+
+        # AND the fill colors are as expected
+        for col in ["Patient ID", "Sex", "Diagnosis", "Component"]:
+            assert sheet1[f"{columns[col]}1"].fill.start_color.index == LIGHT_BLUE
+
+        for col in ["Year of Birth", "Cancer Type", "Family History"]:
+            assert sheet1[f"{columns[col]}1"].fill.start_color.index == GRAY
+
+        for col in ["Patient ID", "Sex", "Diagnosis", "Component"]:
+            assert sheet1[f"{columns[col]}2"].fill.start_color.index == LIGHT_BLUE
+
+        for col in ["Year of Birth", "Cancer Type", "Family History"]:
+            assert sheet1[f"{columns[col]}2"].fill.start_color.index == WHITE
+
+        # AND a copy of the Excel file is saved to the test directory for manual verification
+        if testing_config.manual_test_verification_enabled:
+            workbook.save(
+                os.path.join(
+                    testing_config.manual_test_verification_path,
+                    "CLI_TestManifestCommand_google_sheet_empty_patient.xlsx",
+                )
+            )
+
+    def test_generate_empty_excel_manifest(
+        self, testing_config: ConfigurationForTesting, runner: CliRunner
+    ) -> None:
+        """
+        Tests for:
+        - command has no errors, has exit code 0
+        - command output has excel file creation message
+
+        Tests for google sheet:
+        - drop downs are populated correctly
+        - required fields are marked as “light blue”,
+          while other non-required fields are marked as white.
+        - first row comments are 'TBD'
+        - first row of 'Family History has its own comment
+
+
+        Manual tests:
+        - Open 'CLI_TestManifestCommand_excel_empty_patient.xlsx'
+        - Select 'Diagnosis' to be 'cancer' in the first row:
+            - 'Cancer Type' and 'Family History' cells in first row should be light blue.
+        - Select 'Diagnosis' to NOT be 'cancer' in the first row:
+            - 'Cancer Type' and 'Family History' cells in first row should be white
```

Comment:
What I was doing in some other tests was to do the selection of data ahead of time so no one has to edit the excel files.

On my machine I don't have an office365 subscription so I cannot actually edit any excel files. But I can still open them to verify the content of them. Please update the data in the files before writing to disk.

Reply:
https://github.com/Sage-Bionetworks/schematic/blob/778bf54db9c5b4de0af334c4efe034b3dde0b348/tests/integration/test_manifest_generation.py#L180-L181

Reply:
Will do